pdf_viewer/prefs.config

```diff
@@ -214,6 +214,12 @@ prerender_next_page_presentation 1
 # Highlight on middle clicks when text is selected and no preview is open
 #highlight_middle_click 1
 
+# Use a super fast index for search instead of the mupdf's implementation
+#super_fast_search 1
+
+# Use case-insensitive search
```

Comment:
This is a bit confusing with insensitive vs sensitive in comment and command.. and despite playing with this setting I've only been able to make case-sensitive searching work.